src/classifier_node.py

```
@@ -25,21 +25,26 @@ def on_image(model, class_pub, image_msg):
     # Ask the classifier to infer result
     try:
         result = model.infer(pil_image)
-    except ValueError as e:
+    except Exception as e:
         rospy.logerr('Error getting classification from Triton: %s', e)
-        rospy.logerr(str(e))
         return
```

Maybe change this `return` to `raise` to re-raise the exception so that rospy's default handler applies, e.g., logging the traceback to `rosout`.

---

Done, changed the one further down to raise an error as well instead of just logging